manifests/r/RicoSuter/NSwagStudio/13.13.0.0/RicoSuter.NSwagStudio.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: RicoSuter.NSwagStudio
+PackageVersion: 13.13.0.0
```

Comment:
The mentioned version is broken

Reply:
You may remove this manifest if this version is broken, but unless the latest one is published, someone else will likely re add this one back.

Reply:
I changed this to use the new 13.13.1.0 version that fixes the crash on startup issue. Thanks for letting me know @svrooij.